src/pages/EventAdministration/components/EventEditor.tsx

```diff
@@ -22,6 +22,7 @@ import MarkdownEditor from 'components/inputs/MarkdownEditor';
 import Select from 'components/inputs/Select';
 import SubmitButton from 'components/inputs/SubmitButton';
 import TextField from 'components/inputs/TextField';
+import TimePicker from 'components/inputs/TimePicker';
```

Comment:
Hva er forskjellen på `TimePicker` komponenten og `DatePicker`? De ser veldig like ut for meg. Foreslår å bruke `DatePicker` siden det blir enklere å vedlikeholde en komponent for å velge tid.

Reply:
Ja de ser veldig like ut, men når det kommer til å velge en nedbetalingstid i timer så vil ikke DatePicker fungere. Dermed har jeg laget en egen komponent som heller gir brukeren mulighet til å velge antall timer og minutter med et hh:mm format, og det vil komme opp en urskive der man kan velge nedbetalingstid.